src/runtime/Makefile

```diff
@@ -459,11 +459,11 @@ ifneq (,$(QEMUCMD))
     # The latest OVMF build should be good for both TDX and SNP
     FIRMWAREPATH_NV := $(PREFIXDEPS)/share/ovmf/OVMF.fd
 
-    ifneq (,$(QEMUFW))
-        FIRMWAREPATH := $(PREFIXDEPS)/share/ovmf/$(QEMUFW)
+    ifneq (,$(QEMUFW_ARM64))
+        FIRMWAREPATH := $(PREFIXDEPS)/share/aavmf/$(QEMUFW_ARM64)
     endif
-    ifneq (,$(QEMUFWVOL))
-        FIRMWAREVOLUMEPATH := $(PREFIXDEPS)/share/ovmf/$(QEMUFWVOL)
+    ifneq (,$(QEMUFWVOL_ARM64))
+        FIRMWAREVOLUMEPATH := $(PREFIXDEPS)/share/aavmf/$(QEMUFWVOL_ARM64)
```

Comment:
Sorry, not exactly happy about this one yet.

I would still keep the QEMUFW and QEMUFWVOL, but add a new one for arm (instead of changing this one), and then we can have the aavmf path there.

Reply:
Sure, let me fix that.

Reply:
@kevinzs2048, I've updated it, please, take a look and let me know if it works as expected for your case.

Reply:
Looks great!